malariagen_data/anopheles.py

```diff
@@ -6106,6 +6106,31 @@ def plot_haplotype_clustering(
         df_samples = self.sample_metadata(
             sample_sets=sample_sets, sample_query=sample_query
         )
+
+        debug("Dealing with color")
+        if color and isinstance(color, str):
+            if color not in df_samples.columns and not color.startswith("cohort_"):
+                color = "cohort_" + color
+            if color not in df_samples.columns:
+                raise ValueError(f"{color!r} is not a known cohort set")
+        elif color and isinstance(color, dict):
+            df_samples["color"] = [""] * len(df_samples.index)
```

Comment:
```suggestion
            df_samples["color"] = ""
```

Possible simplification.